internal/search/doc/doc.go

```diff
@@ -6,117 +6,31 @@ import (
 	"time"
 
 	"sigs.k8s.io/yaml"
-
-	"google.golang.org/appengine/search"
-)
-
-const (
-	identifierStr   = "identifier"
-	documentStr     = "document"
-	repoURLStr      = "repo_url"
-	filePathStr     = "file_path"
-	creationTimeStr = "creation_time"
 )
 
-// Represents an unbreakable character stream.
-type Atom = search.Atom
-
-// Implements search.FieldLoadSaver in order to index this representation of a kustomization.yaml
-// file.
 type KustomizationDocument struct {
-	identifiers   []Atom
-	FilePath      Atom
-	RepositoryURL Atom
-	DocumentData  string
-	CreationTime  time.Time
+	DocumentData  string    `json:"document,omitempty"`
+	Kinds         []string  `json:"kinds,omitempty"`
+	Identifiers   []string  `json:"identifiers,omitempty"`
+	Values        []string  `json:"values,omitempty"`
+	FilePath      string    `json:"file_path,omitempty"`
```

Comment:
Use camel case for those fields.
`file_path` --> `filePath`
`repository_url` --> `repositoryUrl`
`creation_time` --> `creationTime`

Reply:
Done.